lib/todolist/tasks/task.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Todolist.Tasks.Task do
+  use Ecto.Schema
+  use Pow.Ecto.Schema
+  alias Todolist.Repo
```

Comment:
More a design discussion for everyone to chip in:

Should we use `Repo` in a model and create helper functions like `get_all` or should we simply use `Repo` in controller?

Pros
1. Less code
2. Trying to avoid the thinking paradigm of OO whereby data and methods are together.

Reply:
1. Ecto doesn't have the concept of `models`. They call them `schemas`.
2. DB operations should never be in controllers
3. we should create functions like `get_all` in the [context module](https://hexdocs.pm/phoenix/contexts.html) for the resource we are trying to manipulate

Reply:
I have moved the code to different file called tasks.ex